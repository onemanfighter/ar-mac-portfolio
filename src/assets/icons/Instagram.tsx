import * as React from 'react';

const InstagramIcon = (props: React.SVGProps<SVGSVGElement>) => {
  return (
    <svg fill="none" viewBox="0 0 15 15" height="1em" width="1em" {...props}>
      <path fill="currentColor" d="M7.5 5a2.5 2.5 0 100 5 2.5 2.5 0 000-5z" />
      <path
        fill="currentColor"
        fillRule="evenodd"
        d="M4.5 0A4.5 4.5 0 000 4.5v6A4.5 4.5 0 004.5 15h6a4.5 4.5 0 004.5-4.5v-6A4.5 4.5 0 0010.5 0h-6zM4 7.5a3.5 3.5 0 117 0 3.5 3.5 0 01-7 0zM11 4h1V3h-1v1z"
        clipRule="evenodd"
      />
    </svg>
  );
};

export { InstagramIcon };
