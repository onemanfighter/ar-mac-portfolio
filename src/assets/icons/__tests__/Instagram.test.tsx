import { render } from '@testing-library/react';
import { InstagramIcon } from '../Instagram';

describe('Instagram', () => {
  it('should render correctly', () => {
    const { container } = render(<InstagramIcon />);

    expect(container).toMatchSnapshot();
  });
});
