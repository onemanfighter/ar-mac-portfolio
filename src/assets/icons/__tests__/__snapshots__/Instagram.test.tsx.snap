// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Instagram should render correctly 1`] = `
<div>
  <svg
    fill="none"
    height="1em"
    viewBox="0 0 15 15"
    width="1em"
  >
    <path
      d="M7.5 5a2.5 2.5 0 100 5 2.5 2.5 0 000-5z"
      fill="currentColor"
    />
    <path
      clip-rule="evenodd"
      d="M4.5 0A4.5 4.5 0 000 4.5v6A4.5 4.5 0 004.5 15h6a4.5 4.5 0 004.5-4.5v-6A4.5 4.5 0 0010.5 0h-6zM4 7.5a3.5 3.5 0 117 0 3.5 3.5 0 01-7 0zM11 4h1V3h-1v1z"
      fill="currentColor"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
